CHANGES.txt

```diff
@@ -28,6 +28,8 @@ Revision 0.2.6, released XX-07-2019
   S/MIME
 - Added openType support for CMS Content Types and CMS Attributes
   in the RFC5652 module
+- Added openType support to RFC 2986 by importing definitions from
+  the RFC 5280 module so that the same maps are used.
```

Comment:
Would it be more backward-compatible if we move the maps from rfc5280 to rfc2986, then import from rfc2986 to rfc5280 and possibly update in rfc5280 to reflect the latest changes?

Reply:
I believe that the current code is backward compatible.  There were no maps in rfc2986.  This way all of the maps are in rfc5280 for the attributes and the algorithm identifiers.

Reply:
Well, I just worry that RFC with lower number imports something from RFC with much higher number. That makes me think that may be at the time of rfc2986 there was some other, contemporary, RFC which provided similar data structures as rfc5280. May be it was RFC3280...

Then may be it would make sense to move as much maps as possible to rfc3280 and make rfc5280 importing from it. That's all for the same of consistency.

Does it make sense? But even then I am OK to merge this PR as is in hope to get back to that refactoring some time later...

WDYT?